### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by emma13z**

### CODE REVIEW

**Comment by emma13z** [comment]
P.S. Didn't manage to display a number alongside the range slider using "oninput". Although checked out some options online and found a few solutions, decided not to add them since didn't quite get how they work :)

**Comment by emma13z** [comment]
image

File: forms.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        <legend>Basic User Information:</legend>
+        <label for="fname">Full Name:</label>
+        <input type="text" id="fname" name="fname" required /><br />
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email" required /><br />
+        <label for="pwd">Password:</label>
+        <input type="password" id="pwd" name="pwd" required />
+        <label for="pwd-con">Confirm Password:</label>
+        <input type="password" id="pwd-con" name="pwd-con" /><br />
+        <span>Gender:</span>
```
Comment (IvanCeban) [inline forms.html]: Technically `span` is fine here. But semantically this is a label of radio buttons. 
So would be better to use a `label` tag here

**Comment by IvanCeban** [comment]
I added one more hint. It might help you resolve the issue with the range value. 
Great intention to find solution on your own. 
In future if you don't know how to resolve an issue, don't hesitate to look for solution in the internet and implement it, even though I didn't explain you this. Any feature can be created in different ways, and you're free to choose your way

File: forms.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        <span>Experience Level with HTML:</span>
+        <label for="exp-level"></label>
+        <input
+          type="range"
+          name="exp-level"
+          id="exp-level"
+          min="1"
+          max="10"
+          oninput="displayValue.value=experience.value"
+        />
```
Comment (IvanCeban) [inline forms.html:97]: Try to use output here

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Great work!

File: forms.html (hunk 2 above)
Comment (emma13z) [reply forms.html:97]: Point taken, changes implemented. Thank you!

**Review by IvanCeban — APPROVED** [review]

**Pull request merged**